src/main/java/com/sigopt/model/Experiment.java

```diff
@@ -62,6 +62,10 @@ public Integer getObservationBudget() {
       return Utils.asInteger(this.get("observation_budget"));
     }
 
+    public Boolean getDevelopment() {
+        return (Boolean) this.get("development");
+    }
```

Comment:
should probably be `boolean` not `Boolean`, as in 757ed8182ced049e266a82c85cdf317db428c83c